src/main/java/apprenticeship/Board.java

```diff
@@ -3,15 +3,41 @@
 import java.util.Arrays;
 
 public class Board {
-    private String[] cells = {"1", "2", "3", "4", "5", "6", "7", "8", "9"};
+    private int boardSize;
```

Comment:
I like how you created this `boardSize` variable. Nitpicking here, maybe make it a `final`?